[1,3]<stderr>:2022-10-18 10:21:48.627912: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
[1,3]<stderr>:To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[1,0]<stderr>:2022-10-18 10:21:48.627906: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
[1,0]<stderr>:To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[1,1]<stderr>:2022-10-18 10:21:48.627906: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
[1,1]<stderr>:To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[1,2]<stderr>:2022-10-18 10:21:48.627906: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
[1,2]<stderr>:To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[1,1]<stderr>:2022-10-18 10:21:50.684783: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30990 MB memory:  -> device: 1, name: Tesla V100-SXM2-32GB, pci bus id: 0000:3b:00.0, compute capability: 7.0
[1,0]<stderr>:2022-10-18 10:21:50.705097: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30990 MB memory:  -> device: 0, name: Tesla V100-SXM2-32GB, pci bus id: 0000:18:00.0, compute capability: 7.0
[1,2]<stderr>:2022-10-18 10:21:50.705517: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30990 MB memory:  -> device: 2, name: Tesla V100-SXM2-32GB, pci bus id: 0000:86:00.0, compute capability: 7.0
[1,3]<stderr>:2022-10-18 10:21:50.724363: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30990 MB memory:  -> device: 3, name: Tesla V100-SXM2-32GB, pci bus id: 0000:af:00.0, compute capability: 7.0
[1,1]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,2]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,3]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,0]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,1]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,2]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,3]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,0]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,2]<stderr>:2022-10-18 10:22:02.235645: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8101
[1,1]<stderr>:2022-10-18 10:22:02.237347: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8101
[1,0]<stderr>:2022-10-18 10:22:02.294450: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8101
[1,3]<stderr>:2022-10-18 10:22:02.306495: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8101
[1,1]<stderr>:2022-10-18 10:22:05.273868: W tensorflow/core/kernels/gpu_utils.cc:50] Failed to allocate memory for convolution redzone checking; skipping this check. This is benign and only means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.
[1,2]<stderr>:2022-10-18 10:22:05.274105: W tensorflow/core/kernels/gpu_utils.cc:50] Failed to allocate memory for convolution redzone checking; skipping this check. This is benign and only means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.
[1,3]<stderr>:2022-10-18 10:22:05.274184: W tensorflow/core/kernels/gpu_utils.cc:50] Failed to allocate memory for convolution redzone checking; skipping this check. This is benign and only means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.
[1,0]<stderr>:2022-10-18 10:22:05.394997: W tensorflow/core/kernels/gpu_utils.cc:50] Failed to allocate memory for convolution redzone checking; skipping this check. This is benign and only means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.
[1,2]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,0]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,1]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,3]<stderr>:WARNING:tensorflow:Model was constructed with shape (None, 108, 2048) for input KerasTensor(type_spec=TensorSpec(shape=(None, 108, 2048), dtype=tf.float32, name='consolidated_input'), name='consolidated_input', description="created by layer 'consolidated_input'"), but it was called on an input with incompatible shape (None, None, None, None).
[1,0]<stderr>:2022-10-18 10:29:34.758225: W tensorflow/python/util/util.cc:368] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
